src/CMakeLists.txt

```diff
@@ -1,24 +1,30 @@
-set(PROJECT_NAME merger)
-set(PROJECT_PREFIX tuple)
+set(LIBSUBDIR tuple)
+set(LIBNAME merger)
+
+set(lua_sources)
+lua_source(lua_sources postload.lua)
+set_property(DIRECTORY PROPERTY ADDITIONAL_MAKE_CLEAN_FILES ${lua_sources})
+
 
 # Add C library
-add_library(${PROJECT_NAME} SHARED
+add_library(${LIBNAME} SHARED
             compat/utils.c
+            ${lua_sources}
             merger/merger.c merger/merger-source.c
 )
 # target_link_libraries(merger ${LUAJIT_LIBRARIES} ${MSGPUCK_LIBRARIES})
-set_target_properties(${PROJECT_NAME} PROPERTIES PREFIX "" OUTPUT_NAME ${PROJECT_NAME})
+set_target_properties(${LIBNAME} PROPERTIES PREFIX "" OUTPUT_NAME ${LIBNAME})
 
 add_subdirectory(msgpuck)
-target_link_libraries(${PROJECT_NAME} msgpuck)
+target_link_libraries(${LIBNAME} msgpuck)
 
 # The dynamic library will be loaded from tarantool executable
 # and will use symbols from it. So it is completely okay to have
 # unresolved symbols at build time.
 if (APPLE)
-    set_target_properties(${PROJECT_NAME} PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
+    set_target_properties(${LIBNAME} PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
 endif()
 
 # Install module
-install(FILES mergerx.lua DESTINATION ${TARANTOOL_INSTALL_LUADIR}/${PROJECT_PREFIX}/)
-install(TARGETS ${PROJECT_NAME} LIBRARY DESTINATION ${TARANTOOL_INSTALL_LIBDIR}/${PROJECT_PREFIX})
+# install(FILES postload.lua DESTINATION ${TARANTOOL_INSTALL_LUADIR}/${LIBSUBDIR}/)
+install(TARGETS ${LIBNAME} LIBRARY DESTINATION ${TARANTOOL_INSTALL_LIBDIR}/${LIBSUBDIR})
```

Comment:
1. postload.lua is bundled, so we don't need to install it separately. We can remove the line.
2. AFAIS, LIBNAME and LIBSUBDIR are not defined.

Reply:
1. Yup, will remove
2. Thy were defined at the top

https://github.com/tsafin/tarantool-merge/blob/6d901c9b87d0dd5a2eca411c67977eae6c32ac47/src/CMakeLists.txt#L1-L2